cmd/kusk/manifests/api_server.yaml

```diff
@@ -129,4 +129,4 @@ spec:
   selector:
     app.kubernetes.io/instance: kusk-gateway-api
     app.kubernetes.io/name: kusk-gateway-api
-  type: LoadBalancer
+  type: ClusterIP
```

Comment:
For some reason I changed this into LoadBalancer in previous commits. Here I reverted it.